### PR TITLE
Jest

### PR DESCRIPTION
**Pull request by pountzas**
- Add tests with jest
- Refactor code

### CODE REVIEW

File: src/add.test.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  removeItem(key) {
+    delete this.store[key];
+  }
+}
+
+document.body.innerHTML = `<input id="taskInput" value="Something">
+    <i class="far fa-calendar-plus"></i>
+    <div id="list"></div>
+    <i class="clearBtn"></i>`;
+
```
Comment (abruzy) [inline src/add.test.js:33]: Kindly fix this error for your add functionality test

Screenshot 2021-07-29 at 17 06 56

File: src/delete.test.js [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+  setItem(key, value) {
+    this.store[key] = String(value);
+  }
+
+  removeItem(key) {
+    delete this.store[key];
+  }
+}
+
+document.body.innerHTML = `<input id="taskInput" value="Something">
```
Comment (abruzy) [inline src/delete.test.js:32]: Kindly fix the error for the delete functionality test

image

**Review by abruzy — CHANGES_REQUESTED** [review]
```markdown
Hi @pountzas & @Jmagero 👋🏽, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/add.test.js (hunk 1 above)
Comment (pountzas) [reply src/add.test.js:33]: It is passing the tests. This error comes from this command location.reload(); if I remove this the app breaks everywhere and can not be fixed.

**Review by carlos-ssh — APPROVED** [review]
Hi @pountzas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**